resources/qml/delegates/EncryptionEnabled.qml

```diff
@@ -0,0 +1,61 @@
+// SPDX-FileCopyrightText: 2023 Nheko Contributors
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+import ".."
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import im.nheko 1.0
+
+Rectangle {
+    id: r
+
+    required property string username
+
+    radius: fontMetrics.lineSpacing / 2 + Nheko.paddingMedium
+    width: parent.width ? parent.width : 0
+    height: contents.implicitHeight + Nheko.paddingMedium * 2
+    color: Nheko.colors.alternateBase
+    border.color: Nheko.theme.green
+    border.width: 2
+
+    RowLayout {
+        id: contents
+
+        anchors.fill: parent
+        anchors.margins: Nheko.paddingMedium
+        spacing: Nheko.paddingMedium
+
+        Image {
+            source: "image://colorimage/:/icons/icons/ui/shield-filled-checkmark.svg?" + Nheko.theme.green
+            Layout.alignment: Qt.AlignVCenter
+            width: 24
+            height: width
+        }
+
+        Column {
+            spacing: Nheko.paddingSmall
+            Layout.fillWidth: true
+
+            MatrixText {
+                text: qsTr("%1 enabled end-to-end encryption").arg(r.username)
+                font.bold: true
+                font.pointSize: 14
+                color: Nheko.colors.text
+                width: parent.width
+            }
+
+            MatrixText {
+                text: qsTr("Encryption keeps your messages safe by locking them with a key that only the people in this room have. "
+                           + "That means that even if somebody gains unauthorized access to your messages, they will not be able to see "
+                           + "what they say.")
```

Comment:
I think all this key stuff is a bit too complicated. How about something like this:

"Encryption keeps your messages safe by only allowing the people you sent the message to to read it. You can further enhance this security by verifying the other parties in a room."

Reply:
Somebody complained that the verification part might encourage people to blindly verify others without understanding why. Do you feel like that is a valid concern? I wanted to put in something about verification but wasn't sure how to word it.

Reply:
Yes, that is something that happens and why I usually discourage people form verifying and instead suggest to rely on TOFU. You could say something like "verify them in real life to ensure, they are the people you wanted to talk to" or something.

Reply:
"You can make sure you are talking to the right person by verifying them in real life"?

Reply:
Sounds good to me

Reply:
In that case, this can be merged at your leisure